tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DatabaseMonitoringPropagator.cs

```diff
@@ -0,0 +1,40 @@
+// <copyright file="DatabaseMonitoringPropagator.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet
+{
+    internal static class DatabaseMonitoringPropagator
+    {
+        private const string SqlCommentSpanService = "dddbs";
+        private const string SqlCommentRootService = "ddps";
+        private const string SqlCommentVersion = "ddpv";
+        private const string SqlCommentEnv = "dde";
+
+        private const string SqlCommentTraceParent = "traceparent";
+
+        internal static string PropagateSpanData(string propagationStyle, string configuredServiceName, Span span)
+        {
+            var propgationComment =
+                $"{SqlCommentRootService}='{configuredServiceName}'," +
+                $"{SqlCommentSpanService}='{span.ServiceName}'," +
+                $"{SqlCommentVersion}='{span.GetTag(Tags.Version)}'," +
+                $"{SqlCommentEnv}='{span.GetTag(Tags.Env)}'";
```

Comment:
Do you know what we are supposed to do if the version or env is not set?

Reply:
I'll check with the DBM team since I just realized that the traces do get the value of  `env:none` if `env` is not set and also can take the `env` value from the Agent but then the comments wouldn't have that.

Reply:
Just checked with them and they confirmed we should:
> if you've got missing values, just omit the missing tags.